src/data/accessibility-quiz.ts

```diff
@@ -248,8 +248,17 @@ const accessibilityQuiz = [
     Link: "https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA"
   },
   {
-    Question:
-      "How can you make focus elements more accessible to screen readers on native mobile apps?",
+    Question: "What is the best way to make your website keyboard accessible?",
+    Answer: 
+    "Don't `<tabindex>` attribute greater than zero",
+    Distractor1: "Use Javascript",
+    Distractor2: "Don't `<tabindex>` attribute greater than zero",
+    Distractor3: "Put the headings in an unordered fashion",
+    Explanation: "Putting the `<tabindex>` attribute greater than zero causes the focus order of a website's page to be disorganized, causing people who only user their keyboards to navigate the web to be confused",
```

Comment:
we should remove the backticks and brackets here

```suggestion
    Explanation: "Putting the tabindex attribute greater than zero causes the focus order of a website's page to be disorganized, causing people who only user their keyboards to navigate the web to be confused",
```